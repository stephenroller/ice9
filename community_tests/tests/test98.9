# if bool -> ; fi											succeed

if true -> 
	write "first"; 
fi

write "second";