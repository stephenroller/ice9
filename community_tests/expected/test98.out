Loading file: generated/test98.tm
first
second
Number of instructions executed = 82
